### PR TITLE
Revert "Revert "Add wrap for ff-nvcodec-headers version 11.1.5.1-0""

### PR DESCRIPTION
**Pull request by nirbheek**
This reverts commit e26eaed80e4de47587aaa24d7963666fca263612.

@eli-schwartz I expect a review from you on this to tell me exactly what you would do differently.

### CODE REVIEW

File: subprojects/ff-nvcodec-headers.wrap [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[wrap-git]
```
Comment (jpakkane) [inline subprojects/ff-nvcodec-headers.wrap:1]: Git wraps are not permitted in wrapdb, only tarballs.

**Comment by jpakkane** [comment]
Strange, the tests should not be passing because `sanity_checks.py` has `self.assertEqual(config.sections()[0], 'wrap-file')`. Maybe it is not being run for new wraps.

File: subprojects/ff-nvcodec-headers.wrap (hunk 1 above)
Comment (nirbheek) [reply subprojects/ff-nvcodec-headers.wrap:1]: ff-nvcodec-headers does not release tarballs, and I do not expect them to ever do that. That's why I added it and pointed it to a tag.

**Comment by nirbheek** [comment]
> Strange, the tests should not be passing because `sanity_checks.py` has `self.assertEqual(config.sections()[0], 'wrap-file')`. Maybe it is not being run for new wraps.

I added a new variable in the sanity checks file called ` PER_PROJECT_PERMITTED_WRAP_GIT` for wraps that are permitted to use git repos since I do not expect this project to ever have a release tarball.

**Comment by nirbheek** [comment]
Actually, I was wrong. There is a tarball release on the github mirror. I will update this to use that.

**Pull request merged**